frontend/src/components/ShootWorkers/WorkerInputGeneric.vue

```diff
@@ -376,4 +422,8 @@ export default {
     flex: 1 1 auto;
     padding: 12px;
   }
+
+  ::v-deep .v-list-item--disabled {
+    opacity:0.5;
+  }
```

Comment:
can this be easily tested so we can be sure that it still works after upgrading vuetify

Reply:
It is possible to write a test that checks if the class exists if you want

Reply:
Okay I tried this but failed. I would suggest to leave this untested for now.

Reply:
I now use `Date.now()`which is certainly cleaner, as it returns a timestamp even if it is not in the format that we usually receive from Gardener.
However, we do not use this value anywhere, as `isValidTerminationDate` is set to false. We do this because we don't know if or when the version expired, but if we don't find it in the cloud profile, we assume it has already expired in the past, therefore we set isValidTerminationDate to false.